src/collective/collectionfilter/interfaces.py

```diff
@@ -176,6 +176,25 @@ class ICollectionFilterSchema(ICollectionFilterBaseSchema):
 class ICollectionSearchSchema(ICollectionFilterBaseSchema):
     """Schema for the search filter.
     """
+    button_text = schema.TextLine(
+        title=_('text_search_button', default=u'Search button text'),
+        description=_(
+            'help_header',
+            u'Text of the search button.'
+        ),
+        required=False,
+        default=_(u'Search')
```

Comment:
I suspect its going to work better if we use "leave blank for default 'search'" as that will allow the portlet to work in multiple languages at once. What do you think @petschki ?